### PR TITLE
OADP-642: spec field descriptions in web console matches yaml case.

### PR DESCRIPTION
**Pull request by kaovilai**
[OADP-642](https://issues.redhat.com//browse/OADP-642)

### CODE REVIEW

**Review by eemcmullan — APPROVED** [review]

File: api/v1alpha1/cloud_storage_types.go [hunk 1]
```diff
@@ -25,18 +25,19 @@ const (
 )
 
 type CloudStorageSpec struct {
-	// Name is the name requested for the bucket (aws, gcp) or container (azure)
+	// `name` is the name requested for the bucket (aws, gcp) or container (azure)
```
Comment (shubham-pampattiwar) [inline api/v1alpha1/cloud_storage_types.go:28]: @kaovilai I am wondering on why do we have strings in backticks ?

Comment (kaovilai) [reply api/v1alpha1/cloud_storage_types.go:28]: lol.. no particular reason.

Comment (shubham-pampattiwar) [reply api/v1alpha1/cloud_storage_types.go:28]: oh ok, lets get rid of the backticks then.

File: api/v1alpha1/cloud_storage_types.go [hunk 2]
```diff
@@ -25,18 +25,19 @@ const (
...
+	// name is the name requested for the bucket (aws, gcp) or container (azure)
 	Name string `json:"name"`
-	// CreationSecret is the secret that is needed to be used while creating the bucket.
+	// creationSecret is the secret that is needed to be used while creating the bucket.
 	CreationSecret corev1.SecretKeySelector `json:"creationSecret"`
-	// EnableSharedConfig enable the use of shared config loading for AWS Buckets
+	// enableSharedConfig enable the use of shared config loading for AWS Buckets
 	EnableSharedConfig *bool `json:"enableSharedConfig,omitempty"`
-	// Tags for the bucket
+	// tags` for the bucket
```
Comment (shubham-pampattiwar) [inline api/v1alpha1/cloud_storage_types.go:34]: NIT: remove trailing backtick

File: api/v1alpha1/cloud_storage_types.go [hunk 3]
```diff
@@ -25,18 +25,19 @@ const (
...
 	// +kubebuilder:validation:Optional
 	Tags map[string]string `json:"tags,omitempty"`
-	// Region for the bucket to be in, will be us-east-1 if not set.
+	// region` for the bucket to be in, will be us-east-1 if not set.
```
Comment (shubham-pampattiwar) [inline api/v1alpha1/cloud_storage_types.go:37]: NIT: remove trailing backtick

**Review by shubham-pampattiwar — CHANGES_REQUESTED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**